### ISSUE TITLE
How to suppress saving to a user-selected directory?

### ISSUE THREAD

**Issue by balamutcz** [mbloch/mapshaper#571]
Hello, please is there some way to suppress the feature **saving output files to a user-selected directory in the web UI**, which was presented in v0.6.16?

Im using mapshaper in Chrome browser, (Win10) and didn't find a way,(parameter?) to force mapshaper to the former behavior. Browser is set to save all downloaded files to the same folder.

Thank you for advice and for all your hard work on project

ps

**Comment by mbloch** [comment]
One option would be to have two buttons on the export menu, one to save the output directly to downloads with one click, like before, and another to save to a user-selected directory (if supported).

Do you just prefer the convenience of saving directly to Downloads, or do you want to prevent saving to a user-selected directory?

**Comment by mbloch** [comment]
The recent change I made was to switch from outputting files as downloads to using the "File System Access API" in supported browsers, which lets users save files a bit more like a desktop app. Now, when you save a file, you are prompted to pick a directory and filename, even if your browser is set to save downloaded files to the downloads folder. If enough users prefer the old way, I can think of a few options. One would be to prompt the user, like this: 

<img width="414" alt="image" src="https://user-images.githubusercontent.com/1232222/212493625-5e339f42-8001-41ac-8867-b48bcc3417d1.png">

I could also add a checkbox to "remember this preference".

Would this work well for you?

**Comment by balamutcz** [comment]
On my opinion the solution with "remember this preference" looks perfect. 

From my point of view should be probably sufficient, if the option can be supressed when processing the -o command in script window...
...when the output command is used in the middle of script, the  "select folder" window is really annoying

thanks ps

**Comment by mbloch** [comment]
I updated the export behavior again -- I think you'll approve of the change. I reverted the default behavior to download files using the browser's download setting, and added a checkbox to the export menu that lets switches over to the other behavior (always prompting for the output destination). This worked better for me than the "remember this preference" solution that we talked about before.

<img width="332" alt="image" src="https://user-images.githubusercontent.com/1232222/212552227-bce82480-256b-4e79-87be-6f490645041c.png">
